staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -4603,8 +4603,10 @@ type NodeSpec struct {
 	// If specified, the node's taints.
 	// +optional
 	Taints []Taint `json:"taints,omitempty" protobuf:"bytes,5,opt,name=taints"`
-	// If specified, the source to get node configuration from
+
+	// Deprecated. If specified, the source to get node configuration from
```

Comment:
```suggestion
	// Deprecated. If specified, the source of the node's configuration.
```

Otherwise it is a very confusing run-on sentence (documentation doesn't necessarily preserve line breaks).

Reply:
The next line also needs a period.

Reply:
I'll pay attention next time. Thank you!